sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/JsonToRow.java

```diff
@@ -98,7 +98,7 @@ public void processElement(ProcessContext context) {
                       context.output(jsonToRow(objectMapper(), context.element()));
                     }
                   }))
-          .setCoder(schema.getRowCoder());
+          .setSchema(schema, SerializableFunctions.identity(), SerializableFunctions.identity());
```

Comment:
This pattern all over the place is kind of annoying. How about adding a `setSchema(schema)` function that uses the new `of(Schema schema)` so we don't have to repeat this pattern all over the place?